src/javascript/lib/landing_pages/ico.js

```diff
@@ -431,3 +436,10 @@ function setupCrowdin() {
         }
     }
 }
+
+function getDocumentUrl(lang = 'en') {
+    if (lang === 'ru') {
```

Comment:
Can you please change this to regex, like `/^(ru)$/i.test(lang)` and use the `lang`var in url? This would make future additions easier to implement.